scripts/chronicle_dump/chronicle_dump.erl

```diff
@@ -150,6 +150,51 @@ sanitize_opt(Value) ->
             {error, invalid_function}
     end.
 
+binarify_output_item(Binary) when is_binary(Binary) -> Binary;
+binarify_output_item(Atom) when is_atom(Atom) -> atom_to_binary(Atom, latin1);
+binarify_output_item(Int) when is_integer(Int) -> integer_to_list(Int);
+binarify_output_item(String) when is_list(String) -> String.
+
+dump_guts(Args) ->
+    {Path, _Options} = parse_args(Args, #{}),
+    Guts = dump_guts_inner(Path),
+    Items = [E || {K, V} <- Guts, E <- [K, V]],
+    ?fmt("~s", [[[binarify_output_item(Item) | <<0:8>>] || Item <- Items]]).
```

Comment:
NIT: the `<<0:8>>` is basically the null-terminator, right? cause the bit syntax is saying give me the 0 value in 8bit syntax.. right? It may be just as easily written as. `<<0>>`:
```
16> <<0>> =:= <<0:8>>.
true
17>
```
Not sure if you prefer the extra verbosity, though.. which I could understand.

Reply:
I patterned this after the existing dump-guts https://src.couchbase.org/source/xref/trinity/ns_server/scripts/dump-guts?r=e2878978#637-638.

IMO, the interactions between the erlang guts scripts and python cbcollect_info using existing patterns are tried and true and was the reason for patterning this change after the existing code.